dcos_installer/upgrade.py

```diff
@@ -74,13 +74,17 @@
 # Check if the node has node/cluster checks and run them
 if [ -f /opt/mesosphere/etc/dcos-diagnostics-runner-config.json ]; then
```

Comment:
While you're touching these two conditional blocks, would you mind indenting them to 4 spaces? Looks like all other indented lines in this file use 4.